source/pidomus.cc

```diff
@@ -997,24 +1003,53 @@ piDoMUS<dim, spacedim, LAC>::solve_jacobian_system(const double /*t*/,
     }
   else
     {
+      unsigned int tot_iteration = 0;
+      try
+        {
+          if (use_double_preconditioner && verbose)
+            pcout << " --> Coarse preconditioner "
+                  << std::endl;
+          PrimitiveVectorMemory<typename LAC::VectorType> mem;
+          SolverControl solver_control (matrices[0]->m(),
```

Comment:
Sì, credo di sì. Ma in assoluto non so come stimarlo. Questo dovrebbe essere un upper bound.


On 25 February 2016 at 23:10:03, Alberto Sartori (notifications@github.com) wrote:

In source/pidomus.cc:

> @@ -997,24 +1003,53 @@ piDoMUS<dim, spacedim, LAC>::solve_jacobian_system(const double /*t*/,
>      }
>    else
>      {
> +      unsigned int tot_iteration = 0;
> +      try
> +        {
> +          if (use_double_preconditioner && verbose)
> +            pcout << " --> Coarse preconditioner "
> +                  << std::endl;
> +          PrimitiveVectorMemory<typename LAC::VectorType> mem;
> +          SolverControl solver_control (matrices[0]->m(),
nel caso di doppio precondizionatore qui non devo fare un numero di iterazioni basso?

—
Reply to this email directly or view it on GitHub.